README.md

```diff
@@ -61,6 +61,25 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 ```
 
+### Autocomplete and Syntax highlighting in IntelliJ IDEA or Android Studio
+You can activate the autocomplete following this additional steps:
+- Install danger on your local machine
+- Go to `Preferences -> Build, Execution, Deployment -> Compiler -> Kotlin Compiler` (`Preferences -> Kotlin Compiler` in Android Studio)
+- At the bottom you will find a section `Kotlin Scripting`
+- Complete the field `Script template classes` with  `com.danger.kotlin.kts.DangerFileScript`
+- Complete the field `Script templates classpath` with `/usr/local/lib/danger/danger-kotlin-kts.jar:/usr/local/lib/danger/kotlin-main-kts.jar:/usr/local/lib/danger/danger-kotlin.jar`
```

Comment:
We don't distribute kotlin-main-kts anymore as separate jar I think, and danger-kotlin-kts should not be needed

Reply:
It looks like the autocomplete in intelliJ still not works without it

Reply:
Without which one of the two?